crates/sui-indexer/src/models/tx_indices.rs

```diff
@@ -25,7 +25,15 @@ pub struct TxDigest {
 
 #[derive(Queryable, Insertable, Selectable, Debug, Clone, Default)]
 #[diesel(table_name = tx_affected_addresses)]
-pub struct StoredTxAffected {
+pub struct StoredTxAffectedAddresses {
+    pub tx_sequence_number: i64,
+    pub affected: Vec<u8>,
+    pub sender: Vec<u8>,
```

Comment:
At some point we should look into how to avoid needing to allocate for these...not a comment on this PR though